hudi-common/src/main/java/org/apache/hudi/common/table/timeline/CompletionTimeQueryView.java

```diff
@@ -223,6 +224,25 @@ public List<String> getStartTimes(
         InstantRange.RangeType.CLOSED_CLOSED, earliestInstantTimeFunc);
   }
 
+  /**
+   * Queries the instant start time with given completion time range.
+   *
+   * @param rangeStart              The query range start completion time.
+   * @param rangeEnd                The query range end completion time.
+   * @param rangeType               The range type.
+   *                                with the minimum completion time.
+   *
+   * @return The sorted instant time list.
+   */
+  @VisibleForTesting
+  public List<String> getStartTimes(
+      String rangeStart,
+      String rangeEnd,
+      RangeType rangeType) {
+    return getStartTimes(metaClient.getCommitsTimeline().filterCompletedInstants(), Option.ofNullable(rangeStart), Option.ofNullable(rangeEnd),
+        rangeType, s -> HoodieInstantTimeGenerator.instantTimeMinusMillis(s, MILLI_SECONDS_IN_ONE_DAY));
```

Comment:
Could we extract `s -> HoodieInstantTimeGenerator.instantTimeMinusMillis(s, MILLI_SECONDS_IN_ONE_DAY)` to a final variable so the assumption is clear?  I think this is also copied from another place.

Reply:
Good idea, I'll convert this to a static `Function`